stable/vulcan/templates/_dogstatsd.tpl

```diff
@@ -0,0 +1,14 @@
+{{- define "dogstatsd-sidecar" -}}
+{{- if .Values.dogstatsd.enabled -}}
+- name: dogstatsd
+  image: datadog/docker-dd-agent:latest-dogstatsd
```

Comment:
I'm not sure which is the difference, but it looks more "natural" for me pick the standalone dogstatsd docker image:
https://hub.docker.com/r/datadog/dogstatsd
And I would pick fixed latest version: `datadog/dogstatsd:7.19.2`
WDYT?

Reply:
Pinned